workers/unity/Assets/Gdk/Core/Error/ForwardingDispatcher.cs

```diff
@@ -0,0 +1,67 @@
+using Improbable.Worker;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
+{
+    public class ForwardingDispatcher : ILogDispatcher
+    {
+        private Connection connection;
+
+#if UNITY_EDITOR
+        private readonly LoggingDispatcher loggingDispatcher = new LoggingDispatcher();
+#endif
+
+        public void SetConnection(Connection connection)
+        {
+            this.connection = connection;
+        }
+
+        public void HandleLog(LogType type, LogEvent logEvent)
+        {
+#if UNITY_EDITOR
+            if (Application.isEditor)
+            {
+                loggingDispatcher.HandleLog(type, logEvent);
+            }
+#endif
+
+            if (connection == null)
+            {
+                return;
+            }
+
+            LogLevel logLevel;
+            var message = logEvent.ToString();
+
+            switch (type)
```

Comment:
Can I suggest using a `private static readonly Dictionary<LogType, LogLevel>` that's initialized inline? May look much more readable.